Sources/Guaka/CommandType/CommandType+Utilities.swift

```diff
@@ -0,0 +1,90 @@
+//
```

Comment:
This is just splitting CommandType to more files, it should not have actual code change